cypress/support/fragments/circulation/newPatronNoticePolicies.js

```diff
@@ -0,0 +1,50 @@
+import getRandomPostfix from '../../utils/stringTools';
+import { Button, TextField, TextArea, NavListItem, Checkbox } from '../../../../interactors';
+
+const actionsButton = Button('Actions');
+const nameField = TextField({ id: 'notice_policy_name' });
+
+export default {
+  defaultUiPatronNoticePolicies : {
+    name: `Test_notice_${getRandomPostfix()}`,
+    description: 'Created by autotest team',
+  },
+  create(patronNoticePolicy) {
+    cy.do(Button({ id: 'clickable-create-entry' }).click());
+    this.fillGeneralInformation(patronNoticePolicy);
+  },
+  fillGeneralInformation: (patronNoticePolicy) => {
+    cy.do([
+      nameField.fillIn(patronNoticePolicy.name),
+      Checkbox({ id: 'notice_policy_active' }).click(),
+      TextArea({ id: 'notice_policy_description' }).fillIn(patronNoticePolicy.description),
+      Button({ id: 'footer-save-entity' }).click(),
+    ]);
+  },
+  check: (patronNoticePolicy) => {
+    cy.expect(NavListItem(patronNoticePolicy.name));
```

Comment:
check your code. you provide name of policy, but try to get name from name into this check. May be my misunderstanding